test/NLog.Extensions.Logging.Tests/Logging/NLogLoggerProviderTests.cs

```diff
@@ -27,16 +25,19 @@ public void CreateLogger_HappyPath_LoggerWithCorrectName()
         public void Dispose_HappyPath_FlushLogFactory()
         {
             // Arrange
-            var provider = new NLogLoggerProvider();
-
-            var internalLogWriter = CaptureInternalLog();
+            var logFactory = new LogFactory();
+            var logConfig = new NLog.Config.LoggingConfiguration(logFactory);
+            logConfig.AddTarget(new NLog.Targets.MemoryTarget("output"));
+            logConfig.AddRuleForAllLevels(new NLog.Targets.Wrappers.BufferingTargetWrapper("buffer", logConfig.FindTargetByName("output")));
+            logFactory.Configuration = logConfig;
+            var provider = new NLogLoggerProvider(null, logFactory);
```

Comment:
Maybe we could add a helper (in abstract base class?) for these lines? 

(To keep tests simple :))

Reply:
Fixed. Not sure if it is simpler